packages/rich-text/src/plugins/List/__tests__/insertListFragment.test.tsx

```diff
@@ -0,0 +1,200 @@
+/** @jsx jsx */
+import { jsx, assertOutput, createTestEditor } from '../../../test-utils';
+
+describe('insert fragment', () => {
+  const tests = [
+    {
+      title: 'text wrapped in li > p',
+      input: (
+        <hul>
+          <hli>
+            <hp>
+              hello <cursor />
+            </hp>
+          </hli>
+        </hul>
+      ),
+      fragment: (
+        <fragment>
+          <hli>
+            <hp>world</hp>
+          </hli>
+        </fragment>
+      ),
+      expected: (
+        <hul>
+          <hli>
+            <hp>hello world</hp>
+          </hli>
+        </hul>
+      ),
+    },
+    {
+      title: 'text wrapped in li > h*',
+      input: (
+        <hul>
+          <hli>
+            <hp>
+              hello <cursor />
+            </hp>
+          </hli>
+        </hul>
+      ),
+      fragment: (
+        <fragment>
+          <hli>
+            <hh1>world</hh1>
+          </hli>
+        </fragment>
+      ),
+      expected: (
+        <hul>
+          <hli>
+            <hp>hello world</hp>
+          </hli>
+        </hul>
+      ),
+    },
+    {
+      title: 'single li with only asset card',
+      input: (
+        <hul>
+          <hli>
+            <hp>
+              hello
+              <cursor />
+              world
```

Comment:
I assume this is equal to `hello<cursor />world` without any soft line breaks, right?

Reply:
Yes. Prettier decided to format it this way